event-dispatcher/src/main/java/io/sk8s/event/dispatcher/JobLauncher.java

```diff
@@ -75,27 +78,35 @@ public void dispatch(String payload, Map<String, Object> headers, FunctionResour
 							.withVolumeMounts(new VolumeMountBuilder()
 								.withMountPath("/output")
 								.withName("messages")
-							.build())
-						.build())
+								.build()
+							)
+							.build()
+						)
 						.withVolumes(new VolumeBuilder()
-							.withName("messages")
-							.withNewHostPath("/messages")
-						.build())
+								.withName("messages")
+								.withNewHostPath("/messages")
+								.build()
+						)
 					.endSpec()
 				.endTemplate()
-			.build())
+				.build()
+			)
 			.done().toString();
+		// @formatter:on
+
 		System.out.println("JOB: " + job);
 	}
 
-	private List<String> resolvePlaceholders(List<String> original, FunctionResource functionResource) {
+	private List<String> resolvePlaceholders(List<String> original, XFunction functionResource) {
```

Comment:
Why not use the `$()` notation that K8S supports if we're ready to eg create an artificial ENV_VAR?
See http://jeffmendoza.github.io/kubernetes/v1.1/docs/design/expansion.html

Reply:
+1 I was planning to look into exactly that... thanks ;)

Reply:
Done via eb6ba1b